pygeoapi/api.py

```diff
@@ -739,6 +740,26 @@ def get_collection_items(self, headers, args, dataset, pathinfo=None):
             LOGGER.error(exception)
             return headers_, 400, json.dumps(exception)
 
+        # processing cql filter query parameter
+        LOGGER.debug('Processing filter parameter')
+
+        try:
+            cql_expression = args.get('filter')
+            if (cql_expression):
+                cql_handler = load_plugin(
+                    'extensions',
+                    {'name': 'CQL', 'cql_expression': cql_expression}
+                )
+                cql_handler.cql_validation()
+
```

Comment:
Isn’t better here to have a dedicated error to catch?

Reply:
I have implemented it